client/src/app/_services/contact.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class ContactService {
+
+    constructor(private http: HttpClient) { }
+
+    send(contactFormName: String, values: Object, token: String): Observable<any> {
+        return this.http.post('/api/contact', { contactFormName: contactFormName, values: values, token: token });
```

Comment:
Use the short version `{ contactFormName, values, token }`

Reply:
Of course!